Woof/Woof/Model/Owner/Owner+CurrentOwner.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+
+extension Owner {
+    /**
+     Handles the operations with current owner.
+     */
+    struct CurrentOwner {
```

Comment:
Correct me if I'm wrong, but currently we have to write the following code:
```swift
let me = Owner.CurrentOwner.get()
print(me.rating)
```

Should the "current owner" be the `static var` that returns an instance of `Owner` (the `UserDefaults` exploits the same idea)?

So we can write:
```swift
print(Owner.current.rating)
```